CONTRIBUTING.md

```diff
@@ -1,3 +1,9 @@
 # Contributing
 
 I would love to see people collaborate with me on this project of making the most advanced cryptocurrency trading bot on the planet. If you'd like to help out, you can always submit pull requests. Or if you'd like to see features, you can always feel free to do so through Github's issues tab.
+
+## Pre-commit
+
+We leverage the [pre-commit](https://pre-commit.com/) framework.
+
+Install git hooks with `pre-commit install` or run the checks manually `pre-commit run`. Passing `-a` to run over all files.
```

Comment:
This part is purely optional as it runs on CI :)

Reply:
It's still a good feature to have :)

Reply:
It's still a great thing to have :)